03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,25 @@
+1. Quickly explain how middleware is used in an Express application.
+    Middleware in an Express application is like a helper that processes requests before they reach your main code.
```

Comment:
Yup! Also often used to centralize logic that you want to run on multiple requests